From 786105b4d02e932e876040bcdf6900ea26a9b58a Mon Sep 17 00:00:00 2001
From: Zhen Chen <chenzhen@rock-chips.com>
Date: Tue, 26 Nov 2019 08:58:58 +0800
Subject: [PATCH] ARM: dts: rk322x: Adapt gpu node for Mali400 device driver
 base on ARM DDK

The source files are under drivers/gpu/arm/mali400.

Change-Id: I4ee1055cf3f630e2d609ab72e26c36daf51cddbb
Signed-off-by: Zhen Chen <chenzhen@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 8b19cdf0f108..5d94a3fe267f 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -750,8 +750,9 @@
 	};
 
 	gpu: gpu@20000000 {
-		compatible = "rockchip,rk3228-mali", "arm,mali-400";
+		compatible = "arm,mali400";
 		reg = <0x20000000 0x10000>;
+
 		interrupts = <GIC_SPI 6 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>,
@@ -759,16 +760,15 @@
 			     <GIC_SPI 4 IRQ_TYPE_LEVEL_HIGH>,
 			     <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>;
 
-		interrupt-names = "gp",
-				  "gpmmu",
-				  "pp0",
-				  "ppmmu0",
-				  "pp1",
-				  "ppmmu1";
-		clocks = <&cru ACLK_GPU>, <&cru ACLK_GPU>;
-		clock-names = "core", "bus";
-		resets = <&cru SRST_GPU_A>;
+		interrupt-names = "Mali_GP_IRQ",
+				  "Mali_GP_MMU_IRQ",
+				  "Mali_PP0_IRQ",
+				  "Mali_PP0_MMU_IRQ",
+				  "Mali_PP1_IRQ",
+				  "Mali_PP1_MMU_IRQ";
+		clocks = <&cru ACLK_GPU>;
 		#cooling-cells = <2>; /* min followed by max */
+		clock-names = "clk_mali";
 		operating-points-v2 = <&gpu_opp_table>;
 		status = "disabled";
 
-- 
2.35.3

